build.sh

```diff
@@ -44,6 +44,7 @@ install() {
   cp -pR library/ $PKG_DATA_DIR
   cp -pR meta/ $PKG_DATA_DIR
   cp -pR tasks/ $PKG_DATA_DIR
+  cp examples/dr_play.yml $PKG_DATA_DIR
```

Comment:
We have this in `doc` directory already, isn't it enought? Or why is it needed also in `usr/share`?

Reply:
I think that to keep it simple we can also use the dr_play from the examples.
So I will just change the dr.conf to point to the example dr_play which is in the doc directory